frontend/src/LandingView/components/SignInSignUp.js

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import styled from 'styled-components';
-// import Button from '../components/SignInUpButton';
 import StyledSignInUp, { TextIMG } from '../styles/SignInSignUpStyled';
-import textLogo from '../../../src/assets/TH_text_filled.svg';
+import textLogo from '../../../src/assets/Sveza_white.svg';
```

Comment:
This will potentially break during build. Please make it relative to inside the `src` folder.